lib/Core/getDataType.ts

```diff
@@ -169,6 +169,12 @@ export default function (): GetDataTypes {
         value: "shp",
         name: i18next.t("core.dataType.shp"),
         extensions: ["zip"]
+      },
+      {
+        value: "assimp",
+        name: i18next.t("core.dataType.assimp-local"),
+        description: i18next.t("core.dataType.assimp-local-description"),
+        extensions: ["zip"]
```

Comment:
Will remove before merging.

Reply:
Why will this part be removed?

Reply:
That's because assimp converted models are not geo-referenced and will be placed automatically at the center of the earth, which is not very useful.

You can still use it from a catalog and specify an `origin`, `rotation` and `scale` for the model.

Reply:
Why is it here then?

Reply:
So that we can test on the CI site. The bugs apparently manifests only when doing file-uploads not when loaded from a catalog.